mobile/app/(app)/report-issue.tsx

```diff
@@ -0,0 +1,240 @@
+  };
+
+  return (
+    <>
+      <Screen
+        preset="scroll"
+        backgroundColor="white"
+        keyboardShouldPersistTaps="never"
+        ScrollViewProps={{
+          stickyHeaderIndices: [0],
+          bounces: false,
+          keyboardShouldPersistTaps: "handled",
+        }}
+      >
+        <Header
+          title="Report new issue"
+          titleColor="white"
+          barStyle="light-content"
+          leftIcon={<Icon icon="chevronLeft" color="white" />}
+          onLeftPress={() => router.back()}
+        />
+        <YStack paddingVertical="$lg" paddingHorizontal="$md">
+          {/* questions container */}
+
+          {/* //TODO: is the polling station required? */}
+          <YStack gap="$lg">
+            <Controller
+              key="polling_station"
+              name="polling_station"
+              control={control}
+              render={({ field: { onChange, value = { id: "", details: "" } } }) => (
+                <>
+                  {/* select polling station */}
+                  <FormElement title="Polling station">
+                    <Select
+                      value={value.id}
+                      options={pollingStations}
+                      placeholder="Select polling station"
+                      onValueChange={(id) => onChange({ ...value, id, details: "" })}
+                      // onOpenChange={(open) => open && Keyboard.dismiss()}
+                    />
+                  </FormElement>
+                  {/* polling station details */}
+                  {value.id === "other" && (
+                    <FormInput
+                      title="Polling station details *"
+                      type="textarea"
+                      placeholder="Please write here some identification details for this polling station (such as address, name, number, etc.)"
+                      value={value.details}
+                      onChangeText={(details) => onChange({ ...value, details })}
+                    />
+                  )}
+                </>
+              )}
+            />
+
+            {/* //TODO: does the '*' need to be red? */}
+            {/* issue title */}
+            <Controller
+              key="issue_title"
+              name="issue_title"
+              control={control}
+              rules={{
+                required: { value: true, message: "This field is required." },
+              }}
+              render={({ field: { onChange, value } }) => (
+                <YStack gap="$xxs">
+                  <FormInput
+                    title="Title of issue *"
+                    placeholder="Write a title for this issue."
+                    type="text"
+                    value={value}
+                    onChangeText={onChange}
+                    borderColor={errors.issue_title && "$red5"}
+                    titleProps={errors.issue_title && { color: "hsl(347, 77%, 40%)" }}
+                  />
+                  {errors.issue_title && (
```

Comment:
Partea asta ar trebuie cumva sa fie parte din FormInput ca errorMessage

Reply:
si toata logica de error border, color sa fie in functie de acel errorMessage prop.